src/scripts/config_for_oss_fuzz.py

```diff
@@ -29,7 +29,7 @@
     "--build-fuzzers=libfuzzer",
     "--build-targets=static",
     "--without-os-features=getrandom,getentropy",
-    "--disable-modules=system_rng,processor_rng",
+    "--disable-modules=system_rng,processor_rng,esdm_rng",
```

Comment:
This shouldn’t be necessary - this is only built if explicitly requested. (Right??)